test/src/test/java/org/springframework/security/test/web/servlet/request/SecurityMockMvcRequestBuildersFormLogoutTests.java

```diff
@@ -96,8 +102,23 @@ public void postProcessorsAreMergedDuringMockMvcPerform() throws Exception {
 
 		MvcResult mvcResult = mockMvc.perform(logout()).andReturn();
 		assertThat(mvcResult.getRequest().getMethod()).isEqualTo(HttpMethod.POST.name());
-
+		assertThat(mvcResult.getRequest().getHeader("Accept"))
+				.isEqualTo(MediaType.toString(Arrays.asList(MediaType.TEXT_HTML, MediaType.ALL)));
+		assertThat(mvcResult.getRequest().getRequestURI()).isEqualTo("/logout");
+		assertCsrfToken(mvcResult.getRequest());
 		verify(postProcessor).postProcessRequest(any());
 	}
 
+	private void assertCsrfToken(MockHttpServletRequest request) {
```

Comment:
Same as above. I think this assertion is a bit different than some of our other tests. We really only need to ensure that the HTTP parameter `_csrf` exists and is not empty.